java/maven/src/org/netbeans/modules/maven/execute/MavenCommandLineExecutor.java

```diff
@@ -1029,7 +1028,6 @@ private File checkAvailability(String ver, VersionRange vr, InputOutput ioput) {
         } else {
             f.mkdirs();
             ioput.getOut().println("NetBeans: Downloading and unzipping Maven version " + ver);
-            ZipInputStream str = null;
```

Comment:
wow, didn't know NB downloads maven - wondering when this happens.

maven is bundled with NB.

Reply:
@mbien take a look at Options / Java / Maven / Experimental.  It's kind of an own take on maven wrapper?!  I suggested in the PR for mvnw #3198 that we might want to consider dropping this support and going all in with the wrapper.

Reply:
yeah I think this feature is a bit questionable. The way maven works: there is really no reason to use anything else except the latest within the same major version. 3.x for example.

and since the introduction of `mvnw` this became redundant too.

so +1 from me too to remove this experimental feature

Reply:
@pepness if you want: feel free to drop the changes in this method since this might disappear at some point.

Reply:
see https://github.com/apache/netbeans/pull/7182 for removal of the mvn downloader